R/ggplotly.R

```diff
@@ -599,7 +599,8 @@ gg2list <- function(p, width = NULL, height = NULL,
         if (xy == "x") {
           ## Facet labels are always on top, so add tick length to move past them
           axisObj[["ticklen"]] <- axisObj[["ticklen"]] + 
-            (unitConvert(stripText, "pixels", type) * 2.5)
+            (unitConvert(stripText, "pixels", type) * 3)
```

Comment:
This is a magic number. I don't think ticklen can vary dynamically so it has to be large enough to accomodate large and small window sizes. It'll be too long for small window sizes but if it's extreme the user can adjust it using `p %>% layout(xaxis=list(ticklen=10), xaxis2=list(ticklen=10), ...)`. 

I'll log an issue on the plotly.js github to see if it would be possible to have ticklen optionally be a proportion of the plot rather than strictly pixels, but otherwise it's a necessary evil